app/population/views/weekly.njk

```diff
@@ -0,0 +1,16 @@
+{% extends "layouts/collection.njk" %}
+
+{% block pageContent %}
+  {% if resultsAsPopulationTables | length  %}
+    <div class="govuk-grid-row">
+      <section class="govuk-grid-column-full govuk-!-margin-bottom-6">
+      {% for table in resultsAsPopulationTables %}
+        {{ govukTable(table) }}
+      {% else %}
+        <h1>No population to display</h1>
```

Comment:
Should this be in the locales for consistency?

Reply:
Yeah, that's a good call.

Reply:
Done!